README.md

```diff
@@ -0,0 +1,27 @@
+# OnlyofficeLoggerHelper
```

Comment:
You should change all mentions of `OnlyofficeLoggerHelper` to `OnlyofficeWebdriverWrapper`